packages/next-swc/crates/core/tests/fixture/optimize_server_react/2/input.js

```diff
@@ -0,0 +1,65 @@
+// https://github.com/vercel/commerce/blob/18167d22f31fce6c90f98912e514243236200989/components/layout/search/filter/dropdown.tsx#L16
+
+'use client'
+
+import { usePathname, useSearchParams } from 'next/navigation'
+import { useEffect, useRef, useState } from 'react'
+
+import { ChevronDownIcon } from '@heroicons/react/24/outline'
+import { FilterItem } from './item'
+
+export default function FilterItemDropdown({ list }) {
+  const pathname = usePathname()
+  const searchParams = useSearchParams()
+  const [active, setActive] = useState('')
+  const [openSelect, setOpenSelect] = useState(false)
+  const ref = useRef < HTMLDivElement > null
```

Comment:
Is usage of `<` and `>` operators intentional? I think that is what it gets parsed at :)

Reply:
Oops, was a mistake that I pasted TS there. Good catch!